### DISCUSSION TITLE
Vacunación responsive móvil

### DISCUSSION THREAD

**Discussion by RamSantoyo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#326] [Show and tell]
![vacunacion](https://user-images.githubusercontent.com/99135741/157536020-fb20a2aa-7323-472f-9a88-4fa3f1a73087.png)
![vacunacion-mobile](https://user-images.githubusercontent.com/99135741/157536051-795b5946-e579-4963-b6de-3a46c4daad98.png)

**Comment by OmarVenturaP** [comment]
Excelente trabajo compañero. Te quedó muy bien. 👍

**Reply by RamSantoyo** [reply]
muchas gracias amigo, en lo que se pueda ayudar aqui estamos

**Comment by JuanSalvador73** [comment]
Excelente trabajo 🗡️

**Reply by RamSantoyo** [reply]
muchas gracias bro
